src/components/navbar/navbar.jsx

```diff
@@ -68,10 +68,9 @@ const Navbar = props => {
             >
               <img
                 alt="logo"
-                height="64x"
                 src="/logo-dsa-adobe-svg.svg"
                 onClick={() => changeRoute("")}
-                style={{ cursor: "pointer" }}
+                style={{ cursor: "pointer" ,height:'auto',maxHeight:'8vh'}}
```

Comment:
use prettier to format the code. for example there are single quotes in the string instead of double quotes.